libs/common/include/launchdarkly/attribute_reference.hpp

```diff
@@ -126,7 +126,7 @@ class AttributeReference {
     /**
      * Default constructs an invalid attribute reference.
      */
-    explicit AttributeReference();
+    AttributeReference();
```

Comment:
Needed to support value-initialization of types when deserializing. I must've been confused when I originally marked this as explicit.